### PR TITLE
Update worker ignition

### PR DESCRIPTION
**Pull request by shhull**
1) Update worker.ign into worker.json, and add the hostname into the json file
2) Update the csr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,10 +79,13 @@ often repeated and in need of automation.
...
     - [Customize when a CMCI module should fail](zos_subsystems/cics/cmci/override_failure)
     - [Resource life cycle and CSD](zos_subsystems/cics/cmci/resource_lifecycle_and_csd)
-- Subsystems
   - [MQ](zos_subsystems/mq)
     - [Interacting with local queues](zos_subsystems/mq/mqsc/qlocal)
 	- [Backing up an application structure](zos_subsystems/mq/mqsc/cfstruct)
+  - [Db2](zos_subsystems/db2)
+    - [Provision a Db2 schema instance](zos_subsystems/db2/db2_schema_provisioning)
+    - [Discover registered Db2 applications](zos_subsystems/db2/db2_discover_applications)
+    - [Discover registered Db2 subsystems](zos_subsystems/db2/db2_discover_subsystems)
```
Comment (bjhuangr) [inline README.md:88]: does these changes related to your PR?

File: zos_subsystems/db2/README.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ of the z/OS system where you installed DOE and <*port*> is the port number that
 For more information: [Integrating Db2 for z/OS database changes into a CI/CD pipeline](https://www.redbooks.ibm.com/Redbooks.nsf/RedpieceAbstracts/redp5646.html?Open)
 
 ## Playbooks Summary
-* [**Provision**](db2_provisioning/db2-schema-provision.yml) - Provision a Db2 schema instance of an application
-* [**Deprovision**](db2_provisioning/db2-schema-deprovision.yml) - Deprovision a Db2 schema instance of an application
-* [**Discovery of Db2 applications**](db2_applications/db2-discover-applications.yml) - Prompt all registered "applications" that can be instantiated
-* [**Discovery of Db2 system information**](db2_discovery/db2-discover-subsystems.yml) - Prompt the information on all registered Db2 systems
+* [**Provision**](db2_schema_provisioning) - Provision a Db2 schema instance of an application
```
Comment (bjhuangr) [inline zos_subsystems/db2/README.md:28]: and those db2 changes ...

**Review by bjhuangr — CHANGES_REQUESTED** [review]
seems you have included changes of other commits, please rebase and re-submit

File: zos_subsystems/db2/README.md (hunk 2 above)
Comment (shhull) [reply zos_subsystems/db2/README.md]: yes, the branch is confusing, I will close this PR and resubmit later.

**Pull request closed without merging**